y3prediction/prediction.py

```diff
@@ -2024,6 +2034,15 @@ def get_production_rates(cv, temperature):
         restart_av_smu = actx.np.zeros_like(restart_cv.mass)
         restart_av_sbeta = actx.np.zeros_like(restart_cv.mass)
         restart_av_skappa = actx.np.zeros_like(restart_cv.mass)
+        temperature_seed = force_evaluation(actx, temperature_seed)
+        restart_av_smu = force_evaluation(actx, restart_av_smu)
+        restart_av_sbeta = force_evaluation(actx, restart_av_sbeta)
+        restart_av_skappa = force_evaluation(actx, restart_av_skappa)
+
+        restart_fluid_state = create_fluid_state(
```

Comment:
doing this twice in a row for some reason?

Reply:
Heh, removed.